### PR TITLE
feat: update wesh init

### PR DESCRIPTION
**Pull request by gfanton**
update weshnet version

### CODE REVIEW

File: go/internal/initutil/orbitdb.go [hunk 1]
```diff
@@ -86,7 +86,7 @@ func (m *Manager) getOrbitDB() (*weshnet.BertyOrbitDB, error) {
 		opts.PubSub = pubsubraw.NewPubSub(node.PubSub, self.ID(), opts.Logger, nil)
 	}
 
-	odb, err := weshnet.NewBertyOrbitDB(m.getContext(), ipfs, opts)
+	odb, err := weshnet.NewWeshOrbitDB(m.getContext(), ipfs, opts)
```
Comment (jefft0) [inline go/internal/initutil/orbitdb.go:89]: When initializing opts, need:

		GroupMetadataStoreType: "berty_group_metadata",
		GroupMessageStoreType:  "berty_group_messages",

File: go.mod [hunk 2]
```diff
@@ -411,6 +411,8 @@ require (
 
 replace (
 	bazil.org/fuse => bazil.org/fuse v0.0.0-20200117225306-7b5117fecadc // specific version for iOS building
+
+	berty.tech/weshnet => ../weshnet
```
Comment (jefft0) [inline go.mod:415]: I am curious. Does this mean that berty will build with whatever version of weshnet that the developer has checked out locally in ../weshnet ?

Comment (D4ryl00) [reply go.mod]: Yes, that allows us to test berty with the local version of weshnet and check that everything is good before pushing weshnet and berty.

**Review by D4ryl00 — APPROVED** [review]
LGTM

**Pull request merged**